src/20240115_Multi_HarmonizeUSDTRiskParametersOnAaveV3Markets/AaveV3Polygon_HarmonizeUSDTRiskParametersOnAaveV3Markets_20240115.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV3PolygonAssets} from 'aave-address-book/AaveV3Polygon.sol';
+import {AaveV3PayloadPolygon} from 'aave-helpers/v3-config-engine/AaveV3PayloadPolygon.sol';
+import {EngineFlags} from 'aave-helpers/v3-config-engine/EngineFlags.sol';
+import {IAaveV3ConfigEngine} from 'aave-helpers/v3-config-engine/IAaveV3ConfigEngine.sol';
+
+/**
+ * @title Harmonize USDT Risk Parameters on Aave V3 Markets
+ * @author Aave Chan Initiative
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0x285c3f9ef8ae3e0286ce18b45dc7605174c3fb61edf1df34ed5d8f5aa621ab9d
+ * - Discussion: https://governance.aave.com/t/arfc-harmonize-usdt-risk-parameters-on-aave-v3-markets/15815
+ */
+contract AaveV3Polygon_HarmonizeUSDTRiskParametersOnAaveV3Markets_20240115 is AaveV3PayloadPolygon {
+  function collateralsUpdates()
+    public
+    pure
+    override
+    returns (IAaveV3ConfigEngine.CollateralUpdate[] memory)
+  {
+    IAaveV3ConfigEngine.CollateralUpdate[]
+      memory collateralUpdate = new IAaveV3ConfigEngine.CollateralUpdate[](1);
+
+    collateralUpdate[0] = IAaveV3ConfigEngine.CollateralUpdate({
+      asset: AaveV3PolygonAssets.USDT_UNDERLYING,
+      ltv: 77_00,
+      liqThreshold: EngineFlags.KEEP_CURRENT,
+      liqBonus: EngineFlags.KEEP_CURRENT,
+      debtCeiling: EngineFlags.KEEP_CURRENT,
```

Comment:
```suggestion
      debtCeiling: 0,
```
@Rozengarden If we wish to remove USDT from isolation mode we should set the debt ceiling of the asset to 0
Same for other networks as well, where USDT is in isolation mode :)

Also I would specify on the writeup, the networks where we are removing USDT from isolation mode.